GCDWebUploader/GCDWebUploader.m

```diff
@@ -126,14 +126,18 @@ - (instancetype)initWithUploadDirectory:(NSString*)path {
 #endif
                      footer = [NSString stringWithFormat:[siteBundle localizedStringForKey:@"FOOTER_FORMAT" value:@"" table:nil], name, version];
                    }
+                     
+                   NSString *keepDirectoryTree = server.keepDirectoryTree ? @"true" : @"false";
```

Comment:
* position is not following code conventions: try running source formatting script